src/content/tutorials/registry/_index.md

```diff
@@ -1,7 +1,6 @@
 ---
 linkTitle: Registry
-title: Registry Guides
-description: Take a look at the list of guides that delve into configuring registries.
```

Comment:
Removed the description, as there is only one tutorial in that section.

Reply:
![grafik](https://github.com/user-attachments/assets/46e9c7d6-512d-4cb8-a1d7-b4ecbca1ed83)

How do we deal with the `description` being mandatory? Something like `description: ""`?